libs/hdf-converters/src/asff-mapper/case-hdf2asff.ts

```diff
@@ -154,6 +154,37 @@ function getCodeForProfileLayer(
   }
 }
 
+function getPassthrough(execTypes: Record<string, unknown>) {
+  //Default behavior for old instances of HDF
+  if (_.has(execTypes, 'Execution.passthrough')) {
+    return _.get(execTypes, 'Execution.passthrough');
+  }
+  //If only one field exists, then is already an object
+  if (_.has(execTypes, 'Execution.passthrough1of1')) {
+    return _.get(execTypes, 'Execution.passthrough1of1');
+  }
+  //Else reconstitute passthrough from detected strings
+  const strStorage = [];
+  if (_.isObject(execTypes.Execution)) {
+    const keyValuePairs = Object.entries(execTypes.Execution).filter(
+      (keyValue) => keyValue[0].startsWith('passthrough')
+    );
+    //No valid passthrough
+    if (keyValuePairs.length <= 1) {
+      return {};
+    }
+    for (const keyValue of keyValuePairs) {
```

Comment:
use destructuring here so const [_key, value] of keyValuePairs to make it clear what you're actually using instead of a random index